### ISSUE TITLE
Import DJ typu "Nelokalizovaná akce"

### ISSUE THREAD

**Issue by motyc** [ARUP-CAS/aiscr-amcr#75]
Při tomto importu logicky pole PIAN u DJ zůstává prázdné, nicméně server pole vyžaduje vyplněné. Import přesto proběhne v pořádku, dojde i k napojení DJ na odpovídající katastr, nicméně v importním nástroji vyskakuje hláška (příklad z amc-test):
`Chyba DB:
ERROR:  column "pian" of relation "dokumentacni_jednotka" does not exist
ŘÁDKA 1: UPDATE dokumentacni_jednotka SET pian = 151418 WHERE id = 87...
                                          ^ in
UPDATE dokumentacni_jednotka SET pian = 151418 WHERE id = 870013`

**Comment by pbartosova** [comment]
Obecně je problém, že server nekontroluje, zda sql příkaz do databáze prošel nebo ne.  Když na něco takového narazím, opravuji to.
Nicméně tato chyba říká, že sloupec v tabulce dokumentacni_jednotka neexistuje. 
Chyba je v návrhu, autor kodu situaci asi špatně pochopil.
V podmínce isNelokalizovana akce Server zjišťuje, do jakého katastru akce patří (pian katastru) a pak se snaží do tabulku dokumentacni_jednotka do sloupce pian uložit id pianu katastru.
Co je tedy správné zadání:
a) lokalizaci resp. pian vůbec neřeším. (zakomentovat kod)
b) do tabulky vazba_pian_parent vložit vazbu nelokalizovaná akce-id pianu katastru.

**Comment by motyc** [comment]
No, správně je B. Ale já mám pocit, že přesně toto se právě stane. V datech jsem vše kontroloval a proběhne to správně, jen se tam objeví ta chybová hláška. Zjevně se to tedy řeší na dvou místech duplicitně nebo nevím.

**Comment by pbartosova** [comment]
Ok. Zkusím doplnit nějaké logy a zjistím pořadí, jak jdou ty operace za sebou.  Odstraním to chybné vkládání do tabulky Dj a podívám se, kdy vkládá údaj do tabulky vazba_pian_parent.
Dostala bych se komu, až po novém roce. Vzhledem k tomu, že to jen váš nástroj a nezpůsobuje to chyby, tak to snad nevadí.

**Comment by motyc** [comment]
Ano, souhlas.
